Update evohome, honeywell markdown with latest changes (#7649)
home-assistant/home-assistant.github.io: source/_components/climate.honeywell.markdown

---

The `honeywell` climate platform let you control Honeywell Connected thermostats from Home Assistant.

<p class='note'>
This platform is actually a hybrid combination of two distinct climate systems based upon either [somecomfort](https://github.com/kk7ds/somecomfort) (for US-based systems), or [evohome-client](https://github.com/watchforstock/evohome-client) (for international/EU systems) - see the `region` parameter, below. Before you proceed, please be clear which client library is appropriate to your system.
</p>

> Although both regions remain support here, the `evohome-client`-based systems have a new [platform](/components/evohome/) available that may suit your needs better.

Reviewer: ```suggestion
Although both regions remain supported here, the `evohome-client`-based systems have a new [component](/components/evohome/) available that may suit your needs better.
```

> To set this climate platform, add the following information to your `configuration.yaml` file:

Reviewer: ```suggestion
To set up this climate platform, add the following information to your `configuration.yaml` file:
```

    scan_interval: 600
```
<p class='note'>

> Scan interval is expressed in seconds. Omitting/mis-configuring `scan_interval` may result in too-frequent polling and cause you to rate-limited by Honeywell.

Reviewer: ```suggestion
Scan interval is expressed in seconds. Omitting or mis-configuring `scan_interval` may result in too-frequent polling and cause you to be rate-limited by Honeywell.
```
